packages/nx/src/command-line/graph/graph.ts

```diff
@@ -1164,3 +1178,27 @@ async function createJsonOutput(
 
   return response;
 }
+
+function getHelpTextFromTarget(
```

Comment:
Are we supporting this for the static web version of the graph app? No right?

Reply:
No that would have to be persisted somehow in `buildEnvironmentJs`. And we'd have to compute every single one of the help commands in the different projects beforehand. So maybe we should try to hide the whole feature in that case?
We can easily check that via `useEnvironmentConfig` and `environment`/`localMode` just like it's done in `graph/shared/src/lib/project-graph-data-service/get-project-graph-data-service.ts`.

Reply:
That compute will be very expensive. We can hide it.

Reply:
Also hiding it for rendering in docs.